acceptance/localcluster/localcluster.go

```diff
@@ -314,6 +316,12 @@ func (l *Cluster) createNodeCerts() {
 	}
 	args := []string{"cert", "--certs=/certs", "create-node", "--key-size=512"}
 	args = append(args, nodes...)
+	if host := os.Getenv("DOCKER_HOST"); host != "" {
+		args = append(args, host)
+	}
+	if runtime.GOOS == "linux" {
+		args = append(args, net.IPv4zero.String())
+	}
```

Comment:
Replace the above with: `args = append(args, dockerIP().String())`. The `DOCKER_HOST` env variable has the format `"tcp://<ip-address>:<port>"`. `dockerIP()` takes care of parsing this for you or returning `0.0.0.0` on linux.